assets/js/components/user-input/UserInputSelectOptions.js

```diff
@@ -178,17 +186,52 @@ export default function UserInputSelectOptions( {
 		}
 
 		return (
-			<div
-				key={ optionSlug }
-				className="googlesitekit-user-input__select-option"
-			>
-				<ListComponent { ...props }>
-					{ options[ optionSlug ] }
-				</ListComponent>
-			</div>
+			'sell_products_or_service' !== optionSlug && (
```

Comment:
We can simplify this, by returning early/skipping when this answer is current index item, instead of wrapping the whole markup in the conditional wrap to all items

```js
if ( 'sell_products_or_service' === optionSlug ) {
     return false;
}
```

Reply:
Thanks @zutigrm resolved in a recent commit.